packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -75,7 +75,52 @@ export type CoinjoinClientAction =
     | ReturnType<typeof clientSessionRoundChanged>
     | ReturnType<typeof clientSessionCompleted>;
 
-const onCoinjoinRoundChanged =
+/**
+ * Show "do not disconnect" screen on Trezor.
+ * Multiple possible setups:
+ * - 1 account on 1 device
+ * - N accounts on 1 devices (like two passphrases)
+ * - N accounts on X devices (like two physical device)
+ */
+export const setBusyScreen =
+    (accountKeys: string[], expiry?: number) => (_dispatch: Dispatch, getState: GetState) => {
+        const {
+            devices,
+            wallet: { accounts },
+        } = getState();
+
+        // collect unique deviceStates from accounts (passphrase)
+        const uniqueDeviceStates = accountKeys.flatMap(key => {
+            const account = accounts.find(a => a.key === key);
+            return account?.deviceState || [];
+        });
+
+        // collect unique physical devices (by device.id)
+        const uniquePhysicalDevices = uniqueDeviceStates.reduce((result, state) => {
+            const device = devices.find(d => d.state === state);
+            if (device && !result.find(d => d.id === device.id)) {
+                return result.concat(device);
+            }
+            return result;
+        }, [] as typeof devices);
+
+        // TODO: check if device is connected, unlocked, and features.busy
+        // async actions on each physical device in sequence
```

Comment:
an utility for this would be nice, but this can be done anytime in the future

Reply:
will be added in upcoming pr